src/server/server.js

```diff
@@ -124,6 +125,16 @@ function processPluginHtml(html, pluginId) {
     });
 }
 
+function stop() {
```

Comment:
I opened a PR for this: https://github.com/albertinad/taco-simulate-server/pull/1